app/javascript/dashboard/routes/dashboard/conversation/contact/NewConversation.vue

```diff
@@ -34,12 +34,17 @@ export default {
   },
   watch: {
     'contact.id'(id) {
-      this.$store.dispatch('contacts/fetchContactableInbox', id);
+      if (id) {
```

Comment:
Doesn't this mean that we are mounting the component even if contact is not available?